src/app/domain/case-viewer/components/questions-panel/questions-panel.component.spec.ts

```diff
@@ -5,7 +5,55 @@ import {CaseViewerModule} from '../../case-viewer.module';
 import {Selector} from '../../../../../../test/selector-helper';
 import {RouterTestingModule} from '@angular/router/testing';
 import {ConfigService} from '../../../../config.service';
-import { mockQuestion } from './mock/mock-questions.mock';
+import {mockQuestion, mockQuestionEmpty, mockQuestions2, mockQuestionsPanelData} from './mock/mock-questions.mock';
+import {api_base_url} from '../../../../enviorment.mock';
+import {Component, CUSTOM_ELEMENTS_SCHEMA, DebugElement, ViewChild} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {PageDateQuestion} from '../../../models/section_fields';
+import {mockActiveRouteQuestionsPanel} from './mock/activeRoute.mock';
+@Component({
+    selector: `app-host-dummy-component`,
+    template: `<app-questions-panel [panelData]="data"></app-questions-panel>`
+})
+class TestQuestionsPanelDummyHostComponent {
+    private data: PageDateQuestion = mockQuestion;
+    @ViewChild(QuestionsPanelComponent)
+    public QuestionsPanelComponent: QuestionsPanelComponent;
+}
+describe('Testing @input', () => {
+    let testHostComponent: TestQuestionsPanelDummyHostComponent;
+    let testHostFixture: ComponentFixture<TestQuestionsPanelDummyHostComponent>;
+    let el: DebugElement;
```

Comment:
Let's use consistent + more verbose, naming conventions around variable names ie.` let element` over `let el` and `let debugElement` over `let de`

Reply:
agreed and amended.